### PR TITLE
Login-signup page added

### PR DESCRIPTION
**Pull request by hit-11**
capture

### CODE REVIEW

**Comment by hit-11** [comment]
Sign up page has also been added 


signup

**Comment by hit-11** [comment]
Sign up page has also been added .

signup

File: views/accounts/login.ejs [hunk 1]
```diff
@@ -1 +1,64 @@
...
+                <div class="form-group">
+                    <label class="checkbox-inline">
+                        <input type="checkbox"/>Rememeber Me
+                    </label>
+                </div>
+                <div class="form-group">
+                    <input type="submit" id="submit" value="Log In" class="btn btn-success btn-block"/>
+                </div>
+                <div class="form-group">
+                    <button type="submit" onclick="window.location.href='/auth/facebook'"  class="btn btn-primary btn-block">LogIn via fB"</button>
```
Comment (AshishkrGoyal) [inline views/accounts/login.ejs]: @hit-11 have you tested it ??
is facebook auth is working properly ??

Comment (hit-11) [reply views/accounts/login.ejs]: error


This is what's showing up !

Comment (hit-11) [reply views/accounts/login.ejs]: Yeah ! thanks no errors ! 👍

**Comment by hit-11** [comment]
@AshishkrGoyal yeah I'll keep that in mind ..

**Pull request merged**